### ISSUE TITLE
Enhance documentation for configuration templates

### ISSUE THREAD

**Issue by bilak** [custom-cards/button-card#388]
**Checklist**
- [ x] I updated the card to the latest version available
- [ x] I cleared the cache of my browser

**Describe the bug**
Hi, I'm trying to use `button-card` with templates. I'm still using dashboards managed by UI (not yaml) so I don't know whether this is also issue or not, but I didn't found any documentation about this. When I configure template within `ui-lovelace.yaml` it's not being loaded. 
Button-card is installed through HACS.

**Version of the card**
Version: v3.3.6

**To Reproduce**
This is the configuration I used:

```yaml
type: 'custom:button-card'
entity: input_boolean.heating_season
template:
  - transparent_button
```

`ui-lovelace.yaml`
```yaml
views: []
button_card_templates:
  transparent_button:
    styles:
      card:
        - background: transparent
```

**Screenshots**
![image](https://user-images.githubusercontent.com/5826062/97226259-08054200-17d4-11eb-8c40-b0ff781c6224.png)
![image](https://user-images.githubusercontent.com/5826062/97226361-29662e00-17d4-11eb-8c6c-38c5dfbb534e.png)

**Desktop (please complete the following information):**
 - Browser safari/chrome (osx)

**Comment by RomRider** [comment]
You have to put them in the raw editor of your dashboard since you are using the ui to configure it. Not in ui-lovelace.yaml

**Comment by bilak** [comment]
@RomRider you can see in first picture that I'm using raw editor for component.

**Comment by RomRider** [comment]
I was speaking about the full config raw editor not the per card one.

**Comment by bilak** [comment]
well I think that doesn't matter, the raw config looks like this (displying only the changed view)

```
title: Home
views:
  - title: ''
    icon: 'mdi:cog'
    path: test
    panel: false
    badges: []
    cards:
      - type: vertical-stack
        cards:
          - type: 'custom:button-card'
            entity: input_boolean.heating_season
            template:
              - transparent_button
```

**Comment by RomRider** [comment]
Exactly, the button-card templates should be put on top of that

**Comment by bilak** [comment]
@RomRider indeed that was the issue. Can we have some note in documentation about this? For someone who is new to this (like me) it could be confusing how to do the configuration properly. So I would add something like:

When configuring templates make sure which type of lovelace you are using. For
* `managed` (changes are done by lovelace ui) - add the template configuration to configuration in raw editor 
    * go to your dashboard
    * click three dots and `Edit dashboard` button
    * click three dots again and click `Raw configuration editor`
* `yaml` - add template configuration to your `ui-lovelace.yaml`

**Comment by RomRider** [comment]
Makes sense I'll add that. However, you can also do a PR against the documentation and I'll merge it 😊

**Comment by bilak** [comment]
will do later today ;)

**Comment by RomRider** [comment]
Fixed in 32e646d58694b1e8891d6de6aecd5821f87d7c31
